cpp/modmesh/buffer/pymod/wrap_SimpleArray.cpp

```diff
@@ -75,15 +75,18 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArray
                             throw std::runtime_error("dtype mismatch");
                         }
                         modmesh::detail::shape_type shape;
+                        modmesh::detail::shape_type stride;
+                        constexpr size_t itemsize = wrapped_type::itemsize();
                         for (ssize_t i = 0; i < arr_in.ndim(); ++i)
                         {
                             shape.push_back(arr_in.shape(i));
+                            stride.push_back(arr_in.strides(i) / itemsize);
```

Comment:
The unit of the `py::array::strides()` is bytes, and the unit of the `SimpleArray::strides()` is the elements.